mssqlcli/mssqlcliclient.py

```diff
@@ -277,7 +278,8 @@ def _exception_found_in(query_response):
 
     @staticmethod
     def _no_results_found_in(query_response):
-        return not query_response.batch_summaries[0].result_set_summaries
+        return not query_response.batch_summaries \
+               or not query_response.batch_summaries[0].result_set_summaries
```

Comment:
The changes in this file fix a bug that causes multi-line statements to crash when multiple statements are batched (i.e. multiple `select` statements in one multi-line query).